### DISCUSSION TITLE
Why do the models look so strange?

### DISCUSSION THREAD

**Discussion by Vasyanator** [seruva19/kubin#171] [General]
What's the point of having symbolic links to files stored in the blobs folder in the model folder rather than files?

**Comment by seruva19** [comment]
I utilized official client for the Huggingface Hub, and that's how it's cache system works. See https://huggingface.co/docs/huggingface_hub/main/guides/manage-cache and
https://huggingface.co/docs/huggingface_hub/main/package_reference/file_download.
There was a discussion concerning using different format (https://github.com/seruva19/kubin/issues/128), but it has not been pursued.
